.standard.yml

```diff
@@ -1,2 +1,5 @@
 plugins:
   - standard-rails
+ignore:
+  - 'db/migrate/*':
+      - Rails/ReversibleMigrationMethodDefinition
```

Comment:
Do not disable this cop please

Reply:
Remove this ignore, but when I try to commit this changes, I have a error for:
standard: Use Ruby Standard Style (https://github.com/standardrb/standard)
  db/migrate/20240214205323_create_active_admin_comments.rb:1:1: Rails/ReversibleMigrationMethodDefinition: Migrations must contain either a `change` method, or both an `up` and a `down` method.
But this migrations are not created by me.

Reply:
Fixed.